downstream/modules/platform/proc-gw-define-rules-triggers.adoc

```diff
@@ -18,32 +18,22 @@ Always:: The trigger should always be fired.
-
-* * * *ends with*: The trigger will see if the value provided by the source ends with the specified *Value* of the trigger. For example, if the source provided a value of ‘John’ the trigger would be True if its *Value* was set to ‘n’ or ‘on’. The trigger would be False if its *Value* was set to ‘z’ because the value ‘John’ coming from the source does not end with the value ’z’ specified by the trigger.
-
-* * * *equal*: The trigger will see if the value provided by the source is equal to (in its entirety) the specified *Value* of the trigger. For example, if the source returned the value ‘John’, the trigger would be True if its Value was set to ‘John’. Any value other than ‘John’ returned from the source would set this trigger to False.
-
-* * * *in*: The *in* condition will see if the value matches one of several values. When in is specified as the *Comparison* the *Value* field can be a comma separated list. For example, if a trigger had a *Value* of ‘John,Donna’ the trigger would be True if the attribute coming from the source had either the value ‘John’ or ‘Donna’ and False otherwize.
-
-* * The value that a users attribute will be matched against based on the *Comparison* field. See examples in the *Comparison* section above for examples. 
-
-NOTE
++
+* *Operation:* This field includes conditional settings that trigger the handling of the rule based on the specified *Groups* criteria. The choices include *and* and *or*. For example, if you select and the user logging in must be a member of all of the groups specified in the *Groups* field for this trigger to be true. Alternatively, if you select *or* the user logging in must be a member of any of the specified *Groups* in order for the trigger to fire. *NOTE*: If you are only keying off one group it doesn’t matter if you select and or or.
+* *Groups:* This is a list of one or more groups coming from the authentication system that the user must be a member of. See the Operation field to determine the behavior of the trigger if more than one group is specified in the trigger.
+* *Attribute:* The map is true or false based on a users attributes coming from the source system.When defining an attribute trigger, the authentication mapping expands to include the following selections:
+* *Operation:* This field includes conditional settings that trigger the handling of the rule based on the specified *Attribute* criteria. In version 2.5.0 this field needs to be set but is ignored. In the future, attribute based maps will allow for the processing of multiple attributes. *NOTE:* If you would like to experiment with multiple attribute maps you can do that through the API but the UI form will remove multi-attribute maps if the authenticator is saved through the UI.
```

Comment:
Did you miss the note formatting in this line or is it deliberate? For what it's worth--notes don't have to be a separate block, we can format notes in-line. See here: https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/#admonition-syntax
According to that it would be NOTE: <text>. 
If this formatting was deliberate, ignore this comment.

Reply:
Fixed.